sdk/communication/communication-sms/test/public/utils/recordedClient.ts

```diff
@@ -62,6 +62,8 @@ export async function createRecorder(context: Test | undefined): Promise<Recorde
     excludedHeaders: [
       "Accept-Language", // This is env-dependent
       "x-ms-content-sha256", // This is dependent on the current datetime
+      "sec-ch-ua" // This is browser dependent 
+      // https://developer.mozilla.org/docs/Web/HTTP/Headers/Sec-CH-UA
```

Comment:
Error being...
```
[browser-tests]   SmsClient [Playback/Record] [AAD]
[browser-tests]     when sending SMS
[browser-tests]       ✖ can send an SMS message
[browser-tests]         Chrome Headless 112.0.5614.0 (Linux x86_64)
[browser-tests]       RecorderError: Unable to find a record for the request POST https://endpoint/sms?api-version=2021-03-07
[browser-tests]       Header differences:
[browser-tests]           <sec-ch-ua> values differ, request <>, record <"Not.A/Brand";v="8", "Chromium";v="114", "HeadlessChrome";v="114">
[browser-tests]       Body differences:
[browser-tests]       
[browser-tests]           at Recorder.handleTestProxyErrors (../../test-utils/recorder/src/recorder.ts:432:13 <- dist-test/index.browser.js:39337:21)
[browser-tests]           at Object.sendRequest (../../test-utils/recorder/src/recorder.ts:456:8 <- dist-test/index.browser.js:39355:24)
[browser-tests]           at async Object.sendRequest (../../core/core-rest-pipeline/src/policies/retryPolicy.ts:60:26 <- dist-test/index.browser.js:35461:34)
[browser-tests]           at async Object.sendRequest (../../core/core-client/src/deserializationPolicy.ts:99:5 <- dist-test/index.browser.js:36943:32)
[browser-tests]           at async Object.sendRequest (../../../common/temp/node_modules/.pnpm/@azure+core-rest-pipeline@1.11.0/node_modules/@azure/core-rest-pipeline/src/policies/bearerTokenAuth
```
More information on the header https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Sec-CH-UA

I'm unsure why the header is empty in the request in playback but not in the recording.
Also, unsure why we have not seen it in other packages.

Nevertheless, ignoring this header seems harmless in this case. 🍏

I would even take a step further and say that this could be added as part of the default sanitizers/matchers as part of the recorder package itself. 🙂

@timovv @scbedd